packages/backend/src/open_payments/payment_pointer/service.ts

```diff
@@ -93,6 +99,7 @@ async function createPaymentPointer(
       assetId: asset.id
     })
     .withGraphFetched('asset')
+    .withGraphFetched('keys')
```

Comment:
ditto below
A separate graphql resolver can be added to fetch a payment pointer's keys (for when `keys` is included in a `paymentPointers` query)
```suggestion
```